src/IFrameNavigator.ts

```diff
@@ -347,11 +352,22 @@ export default class IFrameNavigator implements Navigator {
     private handleResize(): void {
         const selectedView = this.settings.getSelectedView();
         const oldPosition = selectedView.getCurrentPosition();
-        selectedView.goToPosition(oldPosition);
-    }
 
-    private async handleScroll(): Promise<void> {
-        return this.saveCurrentReadingPosition();
+        const fontSize = this.settings.getSelectedFontSize();
+        this.iframe.contentDocument.body.style.fontSize = fontSize;
+        this.iframe.contentDocument.body.style.lineHeight = "1.5";
+
+        const sideMargin = parseInt(fontSize.slice(0, -2)) * 2;
```

Comment:
I talked to Mauricio and Ricardo about this and they thought it was better not to have the margin be independently adjustable. We weren't sure what it should be, but this seemed okay and we can play with it later. Ideally, it would be set to 2rem and we wouldn't need the pixel value, but I need to compute the size of the iframe in pixels and I don't know how else I could do that. The font size is in pixels because all the other proportional measurements in the CSS use that base font size.